packages/ninjam-server/PKGBUILD

```diff
@@ -5,37 +5,44 @@
 _name=ninjam
 pkgname=ninjam-server
 pkgver=0.080
-pkgrel=1
+pkgrel=2
+# git blame -l ninjam/server/ninjamsrv.cpp | grep "#define VERSION" | cut -d ' ' -f 1
+_commit=44c21b98042afc440af456e110f857b7947ff859
 pkgdesc='Ninjam online jamming server'
 arch=(x86_64 aarch64)
 url='https://cockos.com/ninjam/'
-license=(GPL2)
+license=(GPL-2.0-or-later)
 depends=(gcc-libs)
+makedepends=(git)
 groups=(pro-audio)
-conflicts=(ninjam-server-git)
-source=("$pkgname-$pkgver.tar.gz::https://github.com/cbix/$_name/archive/refs/tags/ninjamsrv/v$pkgver.tar.gz"
+source=("git+https://www-dev.cockos.com/ninjam/ninjam.git#commit=$_commit"
         "$_name.service"
         "$_name.sysusers")
-sha256sums=('64a5e2745ed335cc3e750dafde4f5b80405762cbda4def534486ea081adf3eac'
+sha256sums=('4f78057d1f49eb192daed33d2a9c856bf1c56326b2716d35656955bb99450470'
             '3b88687ecfc7ec413bc530ae43a7c13f0924a9a2533c05286903e86bf46dafa0'
             '78a06cd88f3b721237f81436fc8653b5dc6d29ce25eb4ccf1179a51b77b9b4cc')
 backup=("etc/$_name.cfg")
 
+pkgver() {
+  cd $_name/$_name/server
+  sed -nr 's/#define VERSION "v(.+)"$/\1/p' ninjamsrv.cpp
+}
```

Comment:
I don't think we should have a dynamic version in a non-VCS package.

If you use Git sources for a release package, define a tag or commit to use and set the version with the `pkgver` variable as usual.

Reply:
Yea, I agree, just checked the official Arch Linux packages and there is no package using `pkgver()` 👍 9c90ca3b4d0e4f337def9e3dd679b5c0176e03b6